### PR TITLE
[Refactor/#87] : 공동구매 목록 조회 기능 추가 및 UI 수정

### PR DESCRIPTION
**Pull request by juhee067**
## 📌 작업 내용

- tab 정렬 테스트
- 지역 필터링
- 마감, 남은 일수 뱃지 표시
- grid ui 수정

<br/>

## 📌 구현 결과 (선택)

image

### CODE REVIEW

**Comment by JIMIN1020** [comment]
<h2>Coverage Report</h2>
<table> <thead> <tr> <th align="center">Status</th> <th align="left">Category</th> <th align="right">Percentage</th> <th align="right">Covered / Total</th> </tr> </thead> <tbody> <tr> <td align="center">🔵</td> <td align="left">Lines</td> <td align="right">8.72%</td> <td align="right">571 / 6544</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Statements</td> <td align="right">8.72%</td> <td align="right">571 / 6544</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Functions</td> <td align="right">16.96%</td> <td align="right">48 / 283</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Branches</td> <td align="right">26.41%</td> <td align="right">84 / 318</td> </tr> </tbody> </table><details><summary>File Coverage</summary><table> <thead> <tr> <th align="left">File</th> <th align="right">Stmts</th> <th align="right">% Branch</th> <th align="right">% Funcs</th> <th align="right">% Lines</th> <th align="left">Uncovered Lines</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="6"><b>Changed Files</b></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/components/Liquor/GroupBuyingCard.tsx">src/components/Liquor/GroupBuyingCard.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/components/Liquor/GroupBuyingCard.tsx#L1-L5" class="text-red">1-5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/components/Liquor/GroupBuyingCard.tsx#L7-L10" class="text-red">7-10</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/components/Liquor/GroupBuyingCard.tsx#L12-L14" class="text-red">12-14</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/components/Liquor/GroupBuyingCard.tsx#L16-L22" class="text-red">16-22</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/components/Liquor/GroupBuyingCard.tsx#L24-L32" class="text-red">24-32</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/components/Liquor/GroupBuyingCard.tsx#L34" class="text-red">34</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/components/Liquor/GroupBuyingCard.tsx#L36" class="text-red">36</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/components/Liquor/GroupBuyingCard.tsx#L38" class="text-red">38</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/components/Liquor/GroupBuyingCard.tsx#L44" class="text-red">44</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/components/Liquor/GroupBuyingCard.tsx#L46-L48" class="text-red">46-48</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/components/Liquor/GroupBuyingCard.tsx#L51-L53" class="text-red">51-53</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/components/Liquor/GroupBuyingCard.tsx#L56" class="text-red">56</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/components/Liquor/GroupBuyingCard.tsx#L64" class="text-red">64</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/components/Liquor/GroupBuyingCard.tsx#L67" class="text-red">67</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/components/Liquor/GroupBuyingCard.tsx#L81-L83" class="text-red">81-83</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/api/liquor.api.ts">src/features/GroupBuying/api/liquor.api.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/api/liquor.api.ts#L1" class="text-red">1</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/api/liquor.api.ts#L3-L14" class="text-red">3-14</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/api/liquor.api.ts#L16-L17" class="text-red">16-17</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/api/region.api.ts">src/features/GroupBuying/api/region.api.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/api/region.api.ts#L1" class="text-red">1</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/api/region.api.ts#L3-L4" class="text-red">3-4</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/api/region.api.ts#L6-L7" class="text-red">6-7</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Fallback.tsx">src/features/GroupBuying/components/Fallback.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Fallback.tsx#L1" class="text-red">1</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Fallback.tsx#L3" class="text-red">3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Fallback.tsx#L5-L8" class="text-red">5-8</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Fallback.tsx#L10-L11" class="text-red">10-11</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Fallback.tsx#L13" class="text-red">13</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Fallback.tsx#L15" class="text-red">15</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Fallback.tsx#L23-L24" class="text-red">23-24</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Preview.tsx">src/features/GroupBuying/components/Preview.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Preview.tsx#L1-L5" class="text-red">1-5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Preview.tsx#L7-L8" class="text-red">7-8</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Preview.tsx#L10-L26" class="text-red">10-26</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Preview.tsx#L28-L43" class="text-red">28-43</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Preview.tsx#L45" class="text-red">45</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Preview.tsx#L47" class="text-red">47</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Preview.tsx#L49" class="text-red">49</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Preview.tsx#L52" class="text-red">52</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Preview.tsx#L55" class="text-red">55</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Preview.tsx#L69" class="text-red">69</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Preview.tsx#L71-L72" class="text-red">71-72</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Preview.tsx#L75" class="text-red">75</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Preview.tsx#L81" class="text-red">81</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Preview.tsx#L83-L84" class="text-red">83-84</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx">src/features/GroupBuying/components/RegionSelect.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L1-L4" class="text-red">1-4</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L9-L15" class="text-red">9-15</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L17-L18" class="text-red">17-18</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L20-L28" class="text-red">20-28</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L30-L44" class="text-red">30-44</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L46-L47" class="text-red">46-47</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L49-L50" class="text-red">49-50</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L52" class="text-red">52</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L54" class="text-red">54</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L56" class="text-red">56</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L58" class="text-red">58</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L62" class="text-red">62</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L72-L73" class="text-red">72-73</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L75" class="text-red">75</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L78" class="text-red">78</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L86-L87" class="text-red">86-87</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/RegionSelect.tsx#L93-L94" class="text-red">93-94</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx">src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx#L1-L5" class="text-red">1-5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx#L12-L22" class="text-red">12-22</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx#L24-L25" class="text-red">24-25</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx#L27" class="text-red">27</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx#L29" class="text-red">29</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx#L31" class="text-red">31</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Header/Region.tsx">src/features/GroupBuying/components/Header/Region.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Header/Region.tsx#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Header/Region.tsx#L5-L11" class="text-red">5-11</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Header/Region.tsx#L13" class="text-red">13</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Header/Region.tsx#L15" class="text-red">15</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Header/Region.tsx#L17" class="text-red">17</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/components/Header/Region.tsx#L23-L25" class="text-red">23-25</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/hooks/useLiquor.ts">src/features/GroupBuying/hooks/useLiquor.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/hooks/useLiquor.ts#L1-L2" class="text-red">1-2</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/hooks/useLiquor.ts#L4-L23" class="text-red">4-23</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/hooks/useRegion.ts">src/features/GroupBuying/hooks/useRegion.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/hooks/useRegion.ts#L1-L2" class="text-red">1-2</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/GroupBuying/hooks/useRegion.ts#L4-L10" class="text-red">4-10</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/Join/api/signup.api.ts">src/features/Join/api/signup.api.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/Join/api/signup.api.ts#L1-L2" class="text-red">1-2</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/Join/api/signup.api.ts#L5" class="text-red">5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/Join/api/signup.api.ts#L7" class="text-red">7</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/Join/api/signup.api.ts#L9-L11" class="text-red">9-11</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/Join/api/signup.api.ts#L13-L16" class="text-red">13-16</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/Join/api/signup.api.ts#L18-L24" class="text-red">18-24</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/features/Join/api/signup.api.ts#L26-L35" class="text-red">26-35</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/pages/GroupBuyingPage.tsx">src/pages/GroupBuyingPage.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/pages/GroupBuyingPage.tsx#L1-L8" class="text-red">1-8</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/pages/GroupBuyingPage.tsx#L10-L32" class="text-red">10-32</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/pages/GroupBuyingPage.tsx#L34" class="text-red">34</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/pages/GroupBuyingPage.tsx#L36" class="text-red">36</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/pages/GroupBuyingPage.tsx#L38" class="text-red">38</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/pages/GroupBuyingPage.tsx#L43" class="text-red">43</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/pages/GroupBuyingPage.tsx#L47" class="text-red">47</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/pages/GroupBuyingPage.tsx#L53" class="text-red">53</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/pages/GroupBuyingPage.tsx#L59" class="text-red">59</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/routes/index.tsx">src/routes/index.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/routes/index.tsx#L1-L17" class="text-red">1-17</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/routes/index.tsx#L19-L25" class="text-red">19-25</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/routes/index.tsx#L27-L53" class="text-red">27-53</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/routes/index.tsx#L55-L75" class="text-red">55-75</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/routes/index.tsx#L77-L89" class="text-red">77-89</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/routes/index.tsx#L91-L98" class="text-red">91-98</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/routes/index.tsx#L100-L102" class="text-red">100-102</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/efa9236d3495b0b72ff53555d6e036614a8dc3be/src/routes/index.tsx#L104" class="text-red">104</a></td> </tr> </tbody> </table></details>
<em>Generated in workflow <a href=https://github.com/Licruit/Licruit-frontend/actions/runs/10544260083>#172</a></em>

File: src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
-
+interface Props {
+  region?: string | null;
+  sort?: string | null;
+}
+function GroupBuyingGrid({ region, sort }: Props) {
   const { liquorData, fetchNextPage, hasNextPage } = useLiquor(
-    params || 'ranking'
+    sort || 'ranking',
+    region || null
```
Comment (JIMIN1020) [inline src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx:15]: 여기엔 || null 이거 빼도 되지 않을까요?!

File: src/features/GroupBuying/hooks/useRegion.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { useQuery } from '@tanstack/react-query';
+import { getRegion } from '../api/region.api';
+
+export const useRegion = () => {
+  const { data: regionData } = useQuery({
+    queryKey: ['region'],
+    queryFn: getRegion,
+  });
```
Comment (JIMIN1020) [inline src/features/GroupBuying/hooks/useRegion.ts:8]: 이거 한번만 가져오면 되니까 stale 안되게 해야할 것 같아요
refetchOnWindowFocus 이것도 꺼야할 것 같구요

**Review by JIMIN1020 — APPROVED** [review]
좋아요 좋아요

File: src/pages/GroupBuyingPage.tsx [hunk 3]
```diff
@@ -1,20 +1,33 @@
...
-        <GroupBuyingGrid />
+        <Filter>
+          <TabBox>
+            <Tab type='group_buying' queryKey='sort' />
+          </TabBox>
+          <Region />
+        </Filter>
+        <ErrorBoundary FallbackComponent={Fallback} resetKeys={[sort, region]}>
+          <GroupBuyingGrid sort={sort} region={region} />
+        </ErrorBoundary>
```
Comment (SebellKo) [inline src/pages/GroupBuyingPage.tsx:30]: 아 요렇게 내부에서가 아니라 하나더 상위에서 감싸야 하는군요 ~

**Review by SebellKo — APPROVED** [review]
주히님 고생하셨어요 ~ 얼마 안남았어요 화이팅 !

File: src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx (hunk 1 above)
Comment (juhee067) [reply src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx:15]: 얘도 지역이 전체 일 때는 null이어서 넣었습니다!

**Pull request merged**